.github/workflows/build-image.yml

```diff
@@ -33,13 +33,14 @@ jobs:
             latest=auto
           tags: |
             type=edge,branch=main
-            type=match,pattern=v(.*),group=0
+            type=pep440,pattern={{raw}}
+            type=pep440,pattern=v{{major}}.{{minor}}
             type=ref,event=pr
       - uses: docker/build-push-action@v3
         with:
           context: .
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
-          cache-from: type=registry,ref=tootsuite/mastodon:latest
+          cache-from: type=registry,ref=tootsuite/mastodon:edge
```

Comment:
Use the edge tag as cache rather than latest since the tag is usually given from the main branch (edge).